Sources/ArcGISToolkit/Components/BasemapGallery/BasemapGallery.swift

```diff
@@ -214,8 +214,11 @@ extension AlertItem {
     /// - Parameter loadBasemapError: The load basemap error.
     init(loadBasemapError: Error) {
         self.init(
-            title: "Error loading basemap.",
-            message: "\((loadBasemapError as? ArcGISError)?.details ?? "The basemap failed to load for an unknown reason.")"
+            title: String(localized: "Error loading basemap.", bundle: .module),
+            message: String(
+                localized: "\((loadBasemapError as? ArcGISError)?.details ?? "The basemap failed to load for an unknown reason.")",
```

Comment:
Error strings from the SDK/Core won't be localized, so I don't think this will work. Probably need a local string variable and just localized the 2nd bit if it gets there.